### PR TITLE
translation: refine preface index and about_the_book

### PR DESCRIPTION
**Pull request by yuelinxin**
Refined translation for chapter_preface/index.md and about_the_book.md. There are some personal messages in about_the_book where @krahets can modify if deemed necessary.

### CODE REVIEW

File: docs-en/chapter_preface/about_the_book.md [hunk 1]
```diff
@@ -1,45 +1,45 @@
...
-The aim of this project is to create an open source, free, novice-friendly introductory tutorial on data structures and algorithms.
+The aim of this project is to create an open source, free, beginner-friendly crash course on data structures and algorithms.
 
-- Animated graphs are used throughout the book to structure the knowledge of data structures and algorithms in a way that is clear and easy to understand with a smooth learning curve.
-- The source code of the algorithms can be run with a single click, supporting Java, C++, Python, Go, JS, TS, C#, Swift, Rust, Dart, Zig and other languages.
-- Readers are encouraged to help each other and make progress in the chapter discussion forums, and questions and comments can usually be answered within two days.
+- Animated illustrations, structured delivery for data structures and algorithms, with easy-to-understand content, and a smooth learning curve.
+- Run code with just one click, supporting Java, C++, Python, Go, JS, TS, C#, Swift, Rust, Dart, Zig and other languages.
+- Readers are encouraged to help each other in the comment and discussion section for each chapter, questions and comments are usually answered within two days.
```
Comment (krahets) [inline docs-en/chapter_preface/about_the_book.md:7]: in the comment and discussion section for each chapter
|
in the discussion area for each section

File: docs-en/chapter_preface/about_the_book.md [hunk 2]
```diff
@@ -1,45 +1,45 @@
 # The Book
 
-The aim of this project is to create an open source, free, novice-friendly introductory tutorial on data structures and algorithms.
+The aim of this project is to create an open source, free, beginner-friendly crash course on data structures and algorithms.
```
Comment (krahets) [inline docs-en/chapter_preface/about_the_book.md:3]: The aim of this project is to create an open source, free,
|
This open-source project aims to create a free,

File: docs-en/chapter_preface/about_the_book.md [hunk 3]
```diff
@@ -1,45 +1,45 @@
...
+- Animated illustrations, structured delivery for data structures and algorithms, with easy-to-understand content, and a smooth learning curve.
+- Run code with just one click, supporting Java, C++, Python, Go, JS, TS, C#, Swift, Rust, Dart, Zig and other languages.
+- Readers are encouraged to help each other in the comment and discussion section for each chapter, questions and comments are usually answered within two days.
 
 ## Target Readers
 
-If you are a beginner to algorithms, have never touched an algorithm before, or already have some experience brushing up on data structures and algorithms, and have a vague understanding of data structures and algorithms, repeatedly jumping sideways between what you can and can't do, then this book is just for you!
+If you are a beginner to algorithms, you have never learned algorithms before, or you have accumulated some experience in algorithms, but you only have a vague understanding of data structures and algorithms, and you are constantly jumping between "Yep" and "hmm", then this book is for you!
```
Comment (krahets) [inline docs-en/chapter_preface/about_the_book.md:11]: "Yep" and "hmm"
|
"yep" and "hmm"

Should we preserve their consistency?

**Review by krahets — COMMENTED** [review]
Thanks for the great work! I've made some suggestions and please address them.

File: docs-en/chapter_preface/about_the_book.md (hunk 2 above)
Comment (yuelinxin) [reply docs-en/chapter_preface/about_the_book.md:3]: Sorry this is a typo, will fix it

File: docs-en/chapter_preface/about_the_book.md (hunk 3 above)
Comment (yuelinxin) [reply docs-en/chapter_preface/about_the_book.md:11]: Sorry this is a typo, will fix it.

File: docs-en/chapter_preface/about_the_book.md [hunk 4]
```diff
@@ -1,45 +1,45 @@
...
 - Thanks to Tengbao, Qibao, and Feibao for coming up with a creative name for this book that evokes fond memories of writing the first line of code "Hello World!".
 - Thanks to Sutong for designing the beautiful cover and logo for this book and patiently revising it many times under my OCD.
 - Thanks to @squidfunk for writing layout suggestions and for developing the open source documentation theme [Material-for-MkDocs](https://github.com/squidfunk/mkdocs-material/tree/master).
 
-During the writing process, I read many textbooks and articles on data structures and algorithms. These works provide excellent models for this book and ensure the accuracy and quality of its contents. I would like to thank all my teachers and predecessors for their outstanding contributions!
+During the writing process, I read many textbooks and articles on data structures and algorithms. These works provide excellent models for this book and ensure the accuracy and quality of its contents. Thanks for all who came before me for their outstanding contributions!
```
Comment (K3v123) [inline docs-en/chapter_preface/about_the_book.md:41]: "During the writing process, I read many textbooks and articles on data structures and algorithms. These works provide excellent models for this book and ensure the accuracy and quality of its contents. Thanks for all who came before me for their outstanding contributions!" can be changed to ->


"Throughout the writing journey, I delved into numerous textbooks and articles on data structures and algorithms. These resources served as exemplary models, ensuring the accuracy and quality of this book's content. I extend my gratitude to all who preceded me for their invaluable contributions!"

File: docs-en/chapter_preface/about_the_book.md [hunk 5]
```diff
@@ -1,45 +1,45 @@
...
 
-This book promotes a hands-on approach to learning, and in this respect is heavily inspired by ["Hands-On Learning for Depth"](https://github.com/d2l-ai/d2l-zh). I highly recommend this excellent book to all readers.
+This book promotes a hands-on approach to learning, and in this respect is heavily inspired by ["Dive into Deep Learning"](https://github.com/d2l-ai/d2l-zh). I highly recommend this excellent book to all readers.
```
Comment (K3v123) [inline docs-en/chapter_preface/about_the_book.md:43]: "This book promotes a hands-on approach to learning, and in this respect is heavily inspired by ["Dive into Deep Learning"](https://github.com/d2l-ai/d2l-zh). I highly recommend this excellent book to all readers." can be changed/added to ->

"This book advocates for a hands-on learning approach, inspired significantly by ["Dive into Deep Learning"](https://github.com/d2l-ai/d2l-zh). I highly recommend this exceptional book to all readers."

**Review by K3v123 — COMMENTED** [review]
Overall its really good. Some can be simplified a bit. Its quite close to the CN version.
Uh...Might have to double check just in case. I tend to make mistakes unwittingly and might miss something by accident.

**Comment by yuelinxin** [comment]
sorry, i restructured my branches a bit, will address all the issues and make a new PR, thank you for all the reviews.

**Pull request closed without merging**